### ISSUE TITLE
Dateiname der Templates und die CSS-Klassenbezeichnung abgleichen

### ISSUE THREAD

**Issue by zonky2** [contao/core#8564]
es gibt verschiedene Templates in denen tauchen nicht die selben CSS-Klassen als Bezeichnung auf - das macht das "Debugging" bzw. die Suche nach dem passenden Template was ggf. zu ändern ist schwierig...

Beispiel: das Template für das Formular "Passwort vergessen" hat den Dateinamen "mod_password.html5" - die CSS-Klasse ist "mod_lostPassword" - wenn man also anhand des Quelltextes bzw. der CSS-Klasse das Template "mod_lostPassword.html5" sucht oder ändern will, wird man das nicht finden :-(

Eine konsistente Bezeichnung wäre von Vorteil!

**Comment by fritzmg** [comment]
https://github.com/contao/core-bundle/issues/290

**Comment by zonky2** [comment]
@fritzmg - prima... das wäre aber "nur die halbe Miete" => mir geht es ja eher um eine konsistente Bezeichnung zwischen Template- Dateiname und -CSS-Klasse

(ob mit Binnenmajuskel oder nicht, ist mir egal)

**Comment by fritzmg** [comment]
Tja. Kannst du aber im dortigen Issue diskutieren ;)

**Comment by leofeyer** [comment]
Nein, das Issue ist ja schon geschlossen. Und es ging dort um Template-Namen und nicht um CSS-Klassen.

**Comment by zonky2** [comment]
was von den beiden Sachen (Dateiname/CSS-Klasse) angeglichen wird, ist m.E. egal - Hauptsache gleich

**Comment by leofeyer** [comment]
Closing this in favor of https://github.com/contao/contao/issues/3386.
